e0works/MyDiary-front
# feat(view entry): A user should be able to view diary entry

#### What does this PR do?
- implement view single diary entry

#### Description of Task to be completed?
- write an action test
- write reducer test
- add showEntry method in the dashboard

#### How should this be manually tested?
- start the app and log in
- click on the title of a diary entry
- you should see a modal with the diary title and content body

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#161938359](https://www.pivotaltracker.com/story/show/161938359, "view single entry")

#### Screenshots (if appropriate)
<img width="1680" alt="screen shot 2018-11-30 at 11 32 40 pm" src="https://user-images.githubusercontent.com/24890605/49319750-a90ef380-f4fe-11e8-9a01-357fa4b3752c.png">

#### Questions:
